basicLibrary/src/main/java/basicLibrary/Library.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package basicLibrary;
+
+import java.util.Arrays;
+import java.util.Random;
+public class Library {
+    public boolean someLibraryMethod() {
+        return true;
+    }
+
+    public static int[] roll(int n) {
+        // we know die has six sides
+        int numSides = 6;
```

Comment:
When you're hard-coding a value that won't change it's standard practice to use all caps for the variables, separating words with _:
`int NUM_SIDES = 6;`

This makes it easy for other developers to know that it contains a hard-coded value that shouldn't change